dub.json

```diff
@@ -119,6 +122,27 @@
             "mainSourceFile": "source/agora/cli/multi/main.d",
             "excludedSourceFiles": [
                 "source/agora/cli/client/*",
+                "source/agora/cli/checkvtable/*",
+                "source/agora/node/main.d"
+            ]
+        },
+        {
+            "name": "checkvtable",
```

Comment:
As discussed this needs to be split in 2 configurations

Reply:
I fixed it